notebooks/_05-Requirements.md

```diff
@@ -42,7 +42,7 @@ jupyter:
     - Output: Simulated variables 
```

Comment:
Do we know how long the presentation is? That will influence things like expected slide count and how much space we have to work with.

Reply:
Include tests as as requirement along with inputs and outputs.

Reply:
They require causal graphs as inputs to the process too =]

Reply:
Comparisons across methods (causal inference vs traditional DCM) will be done as part of this project. Comparisons along what metrics?

Reply:
I would include plots with posterior-predictive-p-values

Reply:
Please explicitly note:
- the sort of results we expect to present for each simulation and real data example. Be explicit about the
  - goal of each each simulation result we are showcasing. What do we want the viewer to take away?
  - metrics for each result.
  - presentation of metric (bar cart?, table?, infographic?)
  - appropriateness of the metric + presentation. Does it directly showcase what we want the viewer to take away?

Reply:
From Hassan's email:
Presentations are 1 hour long, broken as follows:

20 mins are for presenting
15 mins are for discussants to ask questions
25 mins for general questions